packages/@atjson/offset-annotations/src/annotations/iframe-embed.ts

```diff
@@ -1,11 +1,14 @@
 import { ObjectAnnotation } from "@atjson/document";
-import { CaptionSource } from "./caption-source";
 
 export class IframeEmbed extends ObjectAnnotation<{
   url: string;
   width?: string;
   height?: string;
-  caption?: CaptionSource;
+  /**
+   * Refers to a slice instead of being an
+   * embedded document.
+   */
+  caption?: string;
```

Comment:
Are we cool having attributes named `caption` refer to an ID? We don't want to rename this `captionID` or something similar at this time?

Reply:
EDIT: NVM I see that this is further discussed below

Reply:
This would be weird, since in a react component, you would get `captionID` as a prop that would contain the react tree... which I would not expect.